packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -573,85 +503,71 @@ class ExpansionTile extends StatefulWidget {
   State<ExpansionTile> createState() => _ExpansionTileState();
 }
 
-class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProviderStateMixin {
-  static final Animatable<double> _easeOutTween = CurveTween(curve: Curves.easeOut);
+class _ExpansionTileState extends State<ExpansionTile>
+    with TickerProviderStateMixin, ExpansibleStateMixin<ExpansionTile> {
   static final Animatable<double> _easeInTween = CurveTween(curve: Curves.easeIn);
+  static final Animatable<double> _easeOutTween = CurveTween(curve: Curves.easeOut);
   static final Animatable<double> _halfTween = Tween<double>(begin: 0.0, end: 0.5);
 
   final ShapeBorderTween _borderTween = ShapeBorderTween();
   final ColorTween _headerColorTween = ColorTween();
   final ColorTween _iconColorTween = ColorTween();
   final ColorTween _backgroundColorTween = ColorTween();
-  final Tween<double> _heightFactorTween = Tween<double>(begin: 0.0, end: 1.0);
 
-  late AnimationController _animationController;
   late Animation<double> _iconTurns;
-  late CurvedAnimation _heightFactor;
   late Animation<ShapeBorder?> _border;
   late Animation<Color?> _headerColor;
   late Animation<Color?> _iconColor;
   late Animation<Color?> _backgroundColor;
 
-  bool _isExpanded = false;
-  late ExpansionTileController _tileController;
+  late ExpansionTileThemeData _expansionTileTheme;
   Timer? _timer;
 
+  @override
+  List<Widget> get children => widget.children;
+
+  @override
+  bool get maintainState => widget.maintainState;
+
+  @override
+  bool get initiallyExpanded => widget.initiallyExpanded;
+
+  @override
+  ValueChanged<bool>? get onExpansionChanged => widget.onExpansionChanged;
+
+  @override
+  Duration get expansionDuration => const Duration(milliseconds: 200);
+
+  @override
+  Curve get expansionCurve => Curves.easeIn;
+
+  @override
+  ExpansibleController<ExpansionTile> get controller =>
```

Comment:
Try storing the ExpansionTileController() in another variable. What you might be seeing is that when calling on this getter to assign _state in the mixin, and widget.controller is null, is creates a new ExpansionTileController. Then when you call this getter again, it creates another new one. :)